frontend/components/Icon/Icon.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { styles } from "./styles";
+import * as icons from "../../svgs";
+
+const StyledIcon = styled.i`
```

Comment:
You can just write styledIcon in the styles.js file like everything else.

Reply:
When I do that I get this error ```Syntax Error: `StyledIcon` has already been exported. Exported id
entifiers must be unique. (16:9)```

Reply:
@cartogram this is blocking me now, can't get rid of the error